gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -568,6 +581,8 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
                     host=self._api_endpoint,
                     client_info=client_info,
                 )
+                if CLIENT_LOGGING_SUPPORTED:
+                    _LOGGER.debug(" {{ service.client_name }} client is successfully constructed.")
```

Comment:
Remove this log since this is under the `if "rest_asyncio" in str(transport_init):` code block and there is a separate log for the async client.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2265/commits/c4696b3f307703395ff64582be9b689c632be10f